CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.11)
+project(Venoge VERSION 0.0.0 LANGUAGES CXX)
+
+# Get the correct target locations among all platforms
+include(GNUInstallDirs)
+
+# Target C++11 for now
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
This way is ok, but somewhat buggy/dangerous, I would prefer using `target_compile_features(target PRIVATE cxx_std_11)`.

Reply:
My goal here is to force all the targets inside the project to share the same standard for convenience. But of course it’s possible to switch to a per-target command.

Reply:
I did not know about that. What's the issue with this way, @atussiot?